sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -113,6 +113,16 @@ object functions {
    */
   def lit(literal: Any): Column = typedLit(literal)
 
+  /**
+   * Creates a [[Column]] of literal value.
+   *
+   * An alias of `typedlit`, and it is encouraged to use `typedlit` directly.
+   *
+   * @group normal_funcs
```

Comment:
Could we use `@deprecated` for these aliases?

Reply:
oh I didn't intend to deprecate this (and `count_dictinct`). I will fix PR description.

Reply:
Oh, I see. Yea, I misunderstood it from the word "deprecate" there.